Kp: 1
Ki: 0
Ku: 0.05
edge_x: [0.0,0.5,0.8,0.9,1.0]
edge_y: [0.0,0.5,0.8,0.9,1.0]
coeff: [1.0,1.5,2.0,5.0]
target_x: 10
target_y: 5
target_z: 0